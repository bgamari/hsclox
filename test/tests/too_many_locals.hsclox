{
    var a1 = 1;
    var a2 = 2;
    var a3 = 3;
    var a4 = 4;
    var a5 = 5;
    var a6 = 6;
    var a7 = 7;
    var a8 = 8;
    var a9 = 9;
    var a10 = 10;
    var a11 = 11;
    var a12 = 12;
    var a13 = 13;
    var a14 = 14;
    var a15 = 15;
    var a16 = 16;
    var a17 = 17;
    var a18 = 18;
    var a19 = 19;
    var a20 = 20;
    var a21 = 21;
    var a22 = 22;
    var a23 = 23;
    var a24 = 24;
    var a25 = 25;
    var a26 = 26;
    var a27 = 27;
    var a28 = 28;
    var a29 = 29;
    var a30 = 30;
    var a31 = 31;
    var a32 = 32;
    var a33 = 33;
    var a34 = 34;
    var a35 = 35;
    var a36 = 36;
    var a37 = 37;
    var a38 = 38;
    var a39 = 39;
    var a40 = 40;
    var a41 = 41;
    var a42 = 42;
    var a43 = 43;
    var a44 = 44;
    var a45 = 45;
    var a46 = 46;
    var a47 = 47;
    var a48 = 48;
    var a49 = 49;
    var a50 = 50;
    var a51 = 51;
    var a52 = 52;
    var a53 = 53;
    var a54 = 54;
    var a55 = 55;
    var a56 = 56;
    var a57 = 57;
    var a58 = 58;
    var a59 = 59;
    var a60 = 60;
    var a61 = 61;
    var a62 = 62;
    var a63 = 63;
    var a64 = 64;
    var a65 = 65;
    var a66 = 66;
    var a67 = 67;
    var a68 = 68;
    var a69 = 69;
    var a70 = 70;
    var a71 = 71;
    var a72 = 72;
    var a73 = 73;
    var a74 = 74;
    var a75 = 75;
    var a76 = 76;
    var a77 = 77;
    var a78 = 78;
    var a79 = 79;
    var a80 = 80;
    var a81 = 81;
    var a82 = 82;
    var a83 = 83;
    var a84 = 84;
    var a85 = 85;
    var a86 = 86;
    var a87 = 87;
    var a88 = 88;
    var a89 = 89;
    var a90 = 90;
    var a91 = 91;
    var a92 = 92;
    var a93 = 93;
    var a94 = 94;
    var a95 = 95;
    var a96 = 96;
    var a97 = 97;
    var a98 = 98;
    var a99 = 99;
    var a100 = 100;
    var a101 = 101;
    var a102 = 102;
    var a103 = 103;
    var a104 = 104;
    var a105 = 105;
    var a106 = 106;
    var a107 = 107;
    var a108 = 108;
    var a109 = 109;
    var a110 = 110;
    var a111 = 111;
    var a112 = 112;
    var a113 = 113;
    var a114 = 114;
    var a115 = 115;
    var a116 = 116;
    var a117 = 117;
    var a118 = 118;
    var a119 = 119;
    var a120 = 120;
    var a121 = 121;
    var a122 = 122;
    var a123 = 123;
    var a124 = 124;
    var a125 = 125;
    var a126 = 126;
    var a127 = 127;
    var a128 = 128;
    var a129 = 129;
    var a130 = 130;
    var a131 = 131;
    var a132 = 132;
    var a133 = 133;
    var a134 = 134;
    var a135 = 135;
    var a136 = 136;
    var a137 = 137;
    var a138 = 138;
    var a139 = 139;
    var a140 = 140;
    var a141 = 141;
    var a142 = 142;
    var a143 = 143;
    var a144 = 144;
    var a145 = 145;
    var a146 = 146;
    var a147 = 147;
    var a148 = 148;
    var a149 = 149;
    var a150 = 150;
    var a151 = 151;
    var a152 = 152;
    var a153 = 153;
    var a154 = 154;
    var a155 = 155;
    var a156 = 156;
    var a157 = 157;
    var a158 = 158;
    var a159 = 159;
    var a160 = 160;
    var a161 = 161;
    var a162 = 162;
    var a163 = 163;
    var a164 = 164;
    var a165 = 165;
    var a166 = 166;
    var a167 = 167;
    var a168 = 168;
    var a169 = 169;
    var a170 = 170;
    var a171 = 171;
    var a172 = 172;
    var a173 = 173;
    var a174 = 174;
    var a175 = 175;
    var a176 = 176;
    var a177 = 177;
    var a178 = 178;
    var a179 = 179;
    var a180 = 180;
    var a181 = 181;
    var a182 = 182;
    var a183 = 183;
    var a184 = 184;
    var a185 = 185;
    var a186 = 186;
    var a187 = 187;
    var a188 = 188;
    var a189 = 189;
    var a190 = 190;
    var a191 = 191;
    var a192 = 192;
    var a193 = 193;
    var a194 = 194;
    var a195 = 195;
    var a196 = 196;
    var a197 = 197;
    var a198 = 198;
    var a199 = 199;
    var a200 = 200;
    var a201 = 201;
    var a202 = 202;
    var a203 = 203;
    var a204 = 204;
    var a205 = 205;
    var a206 = 206;
    var a207 = 207;
    var a208 = 208;
    var a209 = 209;
    var a210 = 210;
    var a211 = 211;
    var a212 = 212;
    var a213 = 213;
    var a214 = 214;
    var a215 = 215;
    var a216 = 216;
    var a217 = 217;
    var a218 = 218;
    var a219 = 219;
    var a220 = 220;
    var a221 = 221;
    var a222 = 222;
    var a223 = 223;
    var a224 = 224;
    var a225 = 225;
    var a226 = 226;
    var a227 = 227;
    var a228 = 228;
    var a229 = 229;
    var a230 = 230;
    var a231 = 231;
    var a232 = 232;
    var a233 = 233;
    var a234 = 234;
    var a235 = 235;
    var a236 = 236;
    var a237 = 237;
    var a238 = 238;
    var a239 = 239;
    var a240 = 240;
    var a241 = 241;
    var a242 = 242;
    var a243 = 243;
    var a244 = 244;
    var a245 = 245;
    var a246 = 246;
    var a247 = 247;
    var a248 = 248;
    var a249 = 249;
    var a250 = 250;
    var a251 = 251;
    var a252 = 252;
    var a253 = 253;
    var a254 = 254;
    var a255 = 255;
    var a256 = 256;
}
