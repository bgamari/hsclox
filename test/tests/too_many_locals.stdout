[line 257] Error at 'a256': Too many local variables in function.
